lm_eval/models/optimum_lm.py

```diff
@@ -1,16 +1,25 @@
+import json
+import logging
 from importlib.util import find_spec
 from pathlib import Path
 
 from lm_eval.api.registry import register_model
 from lm_eval.models.huggingface import HFLM
 
 
+logger = logging.getLogger(__name__)
```

Comment:
Could we use the existing lm_eval.utils.eval_logger for this instead perhaps?

Otherwise, this PR lgtm!

Reply:
Thanks! I updated the code.